include/measurement_kit/ooni/error.hpp

```diff
@@ -17,6 +17,7 @@ MK_DEFINE_ERR(MK_ERR_OONI(4), MissingMandatoryKeyError, "")
 MK_DEFINE_ERR(MK_ERR_OONI(5), InvalidMandatoryValueError, "")
 MK_DEFINE_ERR(MK_ERR_OONI(6), MissingRequiredHostError, "")
 MK_DEFINE_ERR(MK_ERR_OONI(7), MissingRequiredUrlError, "")
+MK_DEFINE_ERR(MK_ERR_OONI(8), InvalidGeoIPDatabase, "")
```

Comment:
Could this be `InvalidGeoIpDatabase`? I am old and find it difficult to split words when acronyms and CamelCase are mixed, so I prefer when just one word is capitalized in a acronym.